packages/vue-client/src/router/index.ts

```diff
@@ -31,6 +31,12 @@ const router = createRouter({
       component: () => import("../views/GameView.vue"),
       props: true,
     },
+    {
```

Comment:
I don't believe this is necessary - this file defines the Vue (UI) pages, not the Express (`/api/`) routes.